lib/Dialect/TritonGPU/Transforms/AccelerateMatmul.cpp

```diff
@@ -152,9 +152,16 @@ class BlockedToMFMA : public mlir::RewritePattern {
     auto ctx = oldAType.getContext();
 
     triton::gpu::MfmaEncodingAttr mfmaEnc;
+
+    int64_t nonKDim = 16;
```

Comment:
Note: In this PR we emit **only** mfma instructions with size == 16

When #248 is merged, I'll rebase this PR and turn this back to 32.